public/validation.js

```diff
@@ -0,0 +1,139 @@
+/// add validataion when user enter any youtuber's birthday
+
+// category list of all youtube category
+const categoryList = [
```

Comment:
In the Category List, can you add previous categories that have been used before like Finance, Tech, Kids etc.?
Along with the new categories you have added, we should keep the previous categories too.

Reply:
okey can you send me the previous categories list then i will add all the list in the categories

Reply:
We haven't made any category list before, but you can check the site for the previous entries and see the categories there.

Reply:
Ok, I will look at the website and add as many categories as there are.